shared/components/themes-head/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
My reluctance to do that came from the fact that requiring something in `my-sites` from `layout` felt weird.